documentation/translations/coordinating.rst

```diff
@@ -73,7 +95,8 @@ __ https://github.com/python-docs-translations
 How is a coordinator elected?
 -----------------------------
 
-There is no election. Each translation will sort out the number of coordinators. We recommend 2 or 3 coordinators, though you may begin with one.  Here are some general suggestions.
+There is no election. Each translation will sort out the number of coordinators.
```

Comment:
Instead of "sort out", how about "Each translation team will decide on the number of coordinators."